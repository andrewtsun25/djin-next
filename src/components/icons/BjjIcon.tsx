import { SvgIcon, SvgIconProps } from "@mui/material";
import React from "react";

type BjjIconProps = Omit<SvgIconProps, "viewBox">;

const BjjIcon: React.FC = (props: BjjIconProps) => {
  return (
    <SvgIcon viewBox="0 0 512 512" {...props}>
      <path
        d="M293.293,399.087c3.566,3.242,8.166,4.982,12.852,4.982c3.711,0,7.459-1.071,10.748-3.318l72.561-49.447
			c3.863-2.64,6.637-6.599,7.774-11.141l13.168-52.364c2.17,2.362,3.49,4.198,3.317,5.087l13.685,38.91l-60.11,46.924
			c-4.456,3.48-7.535,8.414-8.692,13.951L331.123,524.05c-2.696,12.929,5.584,25.59,18.503,28.296
			c1.646,0.344,3.3,0.507,4.916,0.507c11.082,0,21.018-7.746,23.38-19.02l25.675-122.812l15.233-11.896l-3.414,6.234
			c-3.949,7.22-3.911,15.96,0.105,23.142l62.663,112.11c4.38,7.832,12.508,12.25,20.895,12.25c3.949,0,7.946-0.976,11.637-3.041
			c11.523-6.436,15.645-21.009,9.199-32.532l-56.208-100.568l38.174-69.749c1.138-2.075,1.816-4.255,2.295-6.455
			c1.052-1.529,1.808-3.289,1.808-5.287v-55.052c0-5.288-3.223-12.383-7.191-15.874l-90.461-80.373
			c-3.969-3.49-12.021-1.884-15.023,2.458l-2.543,4.667c-1.53,1.262-2.936,2.706-4.055,4.475l-50.892,80.574l-58.762,44.083
			c-4.609,3.452-7.124,8.597-7.516,13.913l-16.505-12.269c-3.194-2.372-7.057-3.691-11.044-3.768l-68.697-1.311l17.71-55.959
			l56.772,11.198c1.559,0.306,3.108,0.459,4.647,0.459c10.356,0,19.794-6.77,22.883-17.031l22.511-74.932
			c2.391,5.221,5.584,10.002,9.581,14.019c29.118-12.068,54.507-59.24,60.454-70.982c0.028,0.459,0.144,0.918,0.363,1.358
			c0.88,1.778,3.031,2.486,4.801,1.616c2.82-1.396,4.389-3.729,5.307-6.33c3.185-0.736,9.668,1.979,13.655,4.571
			c1.654,1.081,3.873,0.612,4.963-1.042c1.062-1.625,0.612-3.863-1.042-4.953c-1.846-1.205-9.562-5.91-16.505-5.833
			c-0.086-4.006-0.928-7.487-1.138-8.252c-0.517-1.913-2.486-3.022-4.408-2.506c-1.32,0.364-2.267,1.416-2.554,2.668
			c-0.124,0.555-0.124,1.147,0.038,1.731c0.976,3.529,1.903,11.838-1.511,13.541c-0.306,0.153-0.535,0.383-0.774,0.603
			c-7.793-5.374-17.146-8.654-27.329-8.654c-9.42,0-18.15,2.802-25.59,7.469l3.175-10.596c3.796-12.642-3.376-25.972-16.018-29.777
			c-12.622-3.787-25.971,3.366-29.777,16.017L235.736,220.9l-52.488-10.347l47.822-59.728c6.914-8.635,7-20.894,0.201-29.625
			L144.118,9.224c-8.109-10.414-23.112-12.297-33.545-4.179c-10.413,8.109-12.288,23.122-4.179,33.545l75.573,97.088l-60.674,75.764
			c-2.553,3.194-4.054,6.856-4.743,10.614l-15.242,39.713c-2.209,4.8-4.849,12.88-5.862,18.064l-1.941,15.396l-24.757-34.894
			l7.392-76.481c1.014-10.519-6.684-19.861-17.193-20.875c-10.471-1.052-19.861,6.685-20.875,17.193l-8.081,83.586
			c-0.44,4.581,0.784,9.161,3.433,12.91l48.902,68.936c2.133,3.002,5.011,5.106,8.176,6.445c1.271,0.87,2.735,1.501,4.542,1.521
			l2.353,0.028c0.182,0.01,0.363,0.066,0.555,0.066c0.115,0,0.229-0.048,0.354-0.057l45.747,0.573
			c0.947,0.163,1.894,0.325,2.888,0.345l88.205,1.683l56.648,42.095C292.26,398.657,292.805,398.791,293.293,399.087z
			 M300.006,350.797l61.572-46.196c1.865-1.396,3.452-3.117,4.695-5.088l2.879-4.561l-7.249,28.84l-55.434,37.772l-11.179-8.311
			C296.927,352.633,298.543,351.896,300.006,350.797z"
      />
    </SvgIcon>
  );
};

export { BjjIcon };
