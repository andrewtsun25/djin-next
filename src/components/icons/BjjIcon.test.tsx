import { render } from "@testing-library/react";

import { BjjIcon } from "./BjjIcon";

describe("BjjIcon", () => {
  it("renders", () => {
    const bjjIcon = render(<BjjIcon />);
    expect(bjjIcon).toMatchSnapshot();
  });
});
