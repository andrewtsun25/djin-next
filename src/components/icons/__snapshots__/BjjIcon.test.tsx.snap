// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BjjIcon renders 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 512 512"
      >
        <path
          d="M3628 4557 c-56 -16 -119 -58 -151 -101 -41 -54 -753 -1249 -767 -1286 -9 -25 -16 -200 -24 -641 l-12 -606 -82 -95 c-45 -53 -82 -100 -82 -104 0 -9 269 -614 290 -655 10 -18 20 -13 145 71 154 105 219 171 273 279 36 73 37 77 39 205 2 72 7 410 13 751 l11 620 333 555 c184 305 344 575 357 600 35 67 34 186 -3 256 -65 123 -210 187 -340 151z"
        />
        <path
          d="M3887 3899 c-8 -13 -47 -76 -85 -140 l-69 -115 40 -34 c70 -59 99 -154 73 -244 -13 -45 -37 -80 -117 -177 -55 -66 -98 -123 -95 -126 3 -3 65 -9 138 -14 134 -9 381 -35 451 -47 l38 -7 -390 -323 c-214 -177 -404 -337 -421 -355 -73 -72 -99 -199 -61 -297 16 -42 131 -195 487 -645 256 -324 484 -610 507 -634 61 -64 115 -86 212 -86 71 0 86 3 135 31 61 34 121 105 139 164 19 63 14 159 -12 212 -12 26 -192 262 -400 525 -207 263 -378 481 -380 484 -1 4 189 165 423 358 234 194 444 372 467 396 104 112 179 420 139 575 -45 176 -162 274 -440 369 -248 85 -456 131 -672 147 -89 6 -92 6 -107 -17z"
        />
        <path
          d="M3515 3280 l-186 -309 3 -85 3 -84 210 250 c116 137 220 268 233 290 24 45 29 108 11 155 -10 28 -67 93 -80 93 -5 0 -92 -139 -194 -310z"
        />
        <path
          d="M2497 3304 c-90 -22 -166 -64 -239 -133 -211 -198 -221 -525 -24 -739 45 -49 117 -102 138 -101 7 0 68 67 136 149 l123 149 7 268 8 269 36 71 c34 70 35 72 14 77 -36 10 -139 5 -199 -10z"
        />
        <path
          d="M2441 2317 c-105 -126 -179 -223 -172 -225 7 -2 31 -18 54 -36 42 -32 83 -100 135 -224 l23 -53 68 82 69 83 7 293 c4 162 6 296 4 298 -2 2 -87 -96 -188 -218z"
        />
        <path
          d="M1907 2139 c-83 -44 -119 -157 -78 -242 21 -43 93 -112 105 -100 3 4 6 34 6 67 0 37 8 77 19 101 22 49 86 108 135 126 20 7 36 16 36 20 0 4 -19 17 -43 28 -55 27 -129 27 -180 0z"
        />
        <path
          d="M2087 2029 c-52 -27 -90 -88 -94 -152 -4 -49 5 -71 135 -362 77 -170 139 -311 138 -312 -1 -2 -117 20 -257 47 -379 74 -364 72 -408 60 -107 -28 -169 -153 -127 -253 16 -40 74 -92 118 -105 58 -19 964 -192 1002 -192 52 0 96 21 133 63 37 43 57 105 47 150 -12 60 -434 991 -462 1020 -55 59 -152 74 -225 36z"
        />
        <path
          d="M1931 1705 c-26 -38 -37 -104 -27 -159 9 -46 50 -94 101 -117 40 -19 105 -26 105 -11 0 14 -139 317 -147 319 -4 2 -19 -13 -32 -32z"
        />
        <path
          d="M425 1598 c-149 -28 -306 -151 -370 -292 -97 -210 -57 -436 106 -599 208 -208 531 -207 742 4 366 365 33 984 -478 887z"
        />
        <path
          d="M1655 1521 c-202 -67 -256 -95 -333 -173 -84 -83 -117 -153 -117 -248 0 -97 35 -170 120 -255 162 -161 366 -193 709 -111 82 20 151 37 153 40 2 2 -119 27 -269 56 -150 28 -299 59 -330 67 -111 30 -178 117 -178 231 0 105 54 185 153 227 55 23 100 19 365 -34 133 -26 242 -44 242 -40 0 4 -6 22 -14 40 -14 33 -16 34 -74 35 -110 2 -205 76 -231 179 -6 25 -15 45 -19 44 -4 0 -83 -26 -177 -58z"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 512 512"
    >
      <path
        d="M3628 4557 c-56 -16 -119 -58 -151 -101 -41 -54 -753 -1249 -767 -1286 -9 -25 -16 -200 -24 -641 l-12 -606 -82 -95 c-45 -53 -82 -100 -82 -104 0 -9 269 -614 290 -655 10 -18 20 -13 145 71 154 105 219 171 273 279 36 73 37 77 39 205 2 72 7 410 13 751 l11 620 333 555 c184 305 344 575 357 600 35 67 34 186 -3 256 -65 123 -210 187 -340 151z"
      />
      <path
        d="M3887 3899 c-8 -13 -47 -76 -85 -140 l-69 -115 40 -34 c70 -59 99 -154 73 -244 -13 -45 -37 -80 -117 -177 -55 -66 -98 -123 -95 -126 3 -3 65 -9 138 -14 134 -9 381 -35 451 -47 l38 -7 -390 -323 c-214 -177 -404 -337 -421 -355 -73 -72 -99 -199 -61 -297 16 -42 131 -195 487 -645 256 -324 484 -610 507 -634 61 -64 115 -86 212 -86 71 0 86 3 135 31 61 34 121 105 139 164 19 63 14 159 -12 212 -12 26 -192 262 -400 525 -207 263 -378 481 -380 484 -1 4 189 165 423 358 234 194 444 372 467 396 104 112 179 420 139 575 -45 176 -162 274 -440 369 -248 85 -456 131 -672 147 -89 6 -92 6 -107 -17z"
      />
      <path
        d="M3515 3280 l-186 -309 3 -85 3 -84 210 250 c116 137 220 268 233 290 24 45 29 108 11 155 -10 28 -67 93 -80 93 -5 0 -92 -139 -194 -310z"
      />
      <path
        d="M2497 3304 c-90 -22 -166 -64 -239 -133 -211 -198 -221 -525 -24 -739 45 -49 117 -102 138 -101 7 0 68 67 136 149 l123 149 7 268 8 269 36 71 c34 70 35 72 14 77 -36 10 -139 5 -199 -10z"
      />
      <path
        d="M2441 2317 c-105 -126 -179 -223 -172 -225 7 -2 31 -18 54 -36 42 -32 83 -100 135 -224 l23 -53 68 82 69 83 7 293 c4 162 6 296 4 298 -2 2 -87 -96 -188 -218z"
      />
      <path
        d="M1907 2139 c-83 -44 -119 -157 -78 -242 21 -43 93 -112 105 -100 3 4 6 34 6 67 0 37 8 77 19 101 22 49 86 108 135 126 20 7 36 16 36 20 0 4 -19 17 -43 28 -55 27 -129 27 -180 0z"
      />
      <path
        d="M2087 2029 c-52 -27 -90 -88 -94 -152 -4 -49 5 -71 135 -362 77 -170 139 -311 138 -312 -1 -2 -117 20 -257 47 -379 74 -364 72 -408 60 -107 -28 -169 -153 -127 -253 16 -40 74 -92 118 -105 58 -19 964 -192 1002 -192 52 0 96 21 133 63 37 43 57 105 47 150 -12 60 -434 991 -462 1020 -55 59 -152 74 -225 36z"
      />
      <path
        d="M1931 1705 c-26 -38 -37 -104 -27 -159 9 -46 50 -94 101 -117 40 -19 105 -26 105 -11 0 14 -139 317 -147 319 -4 2 -19 -13 -32 -32z"
      />
      <path
        d="M425 1598 c-149 -28 -306 -151 -370 -292 -97 -210 -57 -436 106 -599 208 -208 531 -207 742 4 366 365 33 984 -478 887z"
      />
      <path
        d="M1655 1521 c-202 -67 -256 -95 -333 -173 -84 -83 -117 -153 -117 -248 0 -97 35 -170 120 -255 162 -161 366 -193 709 -111 82 20 151 37 153 40 2 2 -119 27 -269 56 -150 28 -299 59 -330 67 -111 30 -178 117 -178 231 0 105 54 185 153 227 55 23 100 19 365 -34 133 -26 242 -44 242 -40 0 4 -6 22 -14 40 -14 33 -16 34 -74 35 -110 2 -205 76 -231 179 -6 25 -15 45 -19 44 -4 0 -83 -26 -177 -58z"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
